Lib/inspect.py

```diff
@@ -954,7 +954,12 @@ def getsourcelines(object):
     object = unwrap(object)
     lines, lnum = findsource(object)
 
-    if ismodule(object):
+    if istraceback(object):
+        object = object.tb_frame
+
+    # for module or frame that correspond to module return all source lines
```

Comment:
"... correspond to a module return ..." -> "... corresponds to a module, return ..."